src/main/java/com/externalSorting/app/ExternalSortingRunner.java

```diff
@@ -9,60 +9,74 @@
     private final String originalFile;
     private final String outputFolder;
 
-    public ExternalSortingRunner(String originalFile, String outputFile) {
+    /**
+     * Constructor
+     *
+     * @param originalFile
+     * Path to the file that will be externally sorted
+     * @param outputFolder
+     * Path to the folder which will contain both the temporary files and the output file
+     */
+    public ExternalSortingRunner(String originalFile, String outputFolder) {
         this.originalFile = originalFile;
-        this.outputFolder = outputFile;
+        this.outputFolder = outputFolder;
     }
 
-    protected void externalSort() {
-
-        System.out.println(outputFolder + "/tmpdir");
-        File newDirectory = new File("src/main/resources/tmpdir/");
-        List<File> files = readAndSplitData("src/main/resources/SmallLibrary.txt", newDirectory);
-
-        System.out.println(outputFolder + "/Output.txt");
-        mergeFiles(files, new File("src/main/resources/SmallLibraryOutput.txt"), comparator);
+    /**
+     * This method is meant to be called from main function.
+     * It is responsible for:
+     *  creating a temporary folder which will hold data files
+     *  calling readAndSplitData and mergeFiles
+     *  deleting temporary folder and files after the process is finished
+     */
+    public void externalSort() {
+        File newDirectory = new File(outputFolder + "/tmpdir");
```

Comment:
When formatting String objects, it can be useful to use:
`String.format("%s/tmpdir", outputFolder)`